src/core/songloader.cpp

```diff
@@ -674,21 +670,6 @@ bool SongLoader::LoadRemotePlaylist(const QUrl& url) {
     return false;
   }
 
-  // Save them to a temporary file...
-  QString playlist_filename =
-      Utilities::SaveToTemporaryFile(data_reply->readAll());
-  if (playlist_filename.isEmpty()) {
-    qLog(Error) << url.toString()
-                << "could not write contents to temporary file";
-    return false;
-  }
-
-  qLog(Debug) << url.toString() << "with MIME" << mime_type << "loading from"
-              << playlist_filename;
```

Comment:
We might want to keep the url and mime_type parts of this log message.

Reply:
OK. Added the log entry, and clang-formatted the file. Thanks for showing me how.